### PR TITLE
add ahem font to system for running tests

### PR DESCRIPTION
**Pull request by bobholt**
Add W3C Ahem font to system for running web platform tests

### CODE REVIEW

File: roles/wpt/tasks/main.yml [hunk 1]
```diff
@@ -41,3 +41,22 @@
...
+  get_url:
+    url: http://www.w3.org/Style/CSS/Test/Fonts/Ahem/AHEM____.TTF
+    dest: /usr/local/share/fonts/truetype/web-platform
+    group: root
+    owner: root
+    mode: 0755
+  become: yes
+
+- name: Rebuild font cache
+  command: fc-cache
```
Comment (jugglinmike) [inline roles/wpt/tasks/main.yml]: How about:

```diff
     mode: 0755
   become: yes
+  register: download_ahem

 - name: Rebuild font cache
   command: fc-cache
+  when: "{{download_ahem.changed}}"
```

File: roles/wpt/tasks/main.yml [hunk 2]
```diff
@@ -41,3 +41,22 @@
...
+    state: directory
+    mode: 0775
+  become: yes
+
+- name: Download the Ahem font
+  get_url:
+    url: http://www.w3.org/Style/CSS/Test/Fonts/Ahem/AHEM____.TTF
+    dest: /usr/local/share/fonts/truetype/web-platform
+    group: root
+    owner: root
```
Comment (jugglinmike) [inline roles/wpt/tasks/main.yml]: I believe you get the owner/group for free when you use `become`

Comment (jugglinmike) [inline roles/wpt/tasks/main.yml]: Just verified that.

Comment (bobholt) [inline roles/wpt/tasks/main.yml]: Ah, of course.

**Comment by bobholt** [comment]
Updated with those 2 bits of feedback.

**Comment by jugglinmike** [comment]
This is great, Bob! I just discovered this requirement last night and was thinking of writing a similar patch. You beat me to the :boxing_glove:

**Pull request merged**